OxfordBodleian/Juel-Jensen/BDLaethg28.xml

```diff
@@ -135,9 +138,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </dimensions>
                                     <note>The characters per line are 10–12.</note>
                                     <ab type="pricking">Ruling and pricking are barely visible.</ab>
-                                    <ab type="ruling" subtype="pattern">
+                                    <!--<ab type="ruling" subtype="pattern">
                                         <locus from="2r" to="15v"/>Ruling pattern:
-                                        1A-1A/0-0/0-0/C.</ab>
+                                        1A-1A/0-0/0-0/C.</ab> I don't see the the horizontal ruled lines, do you?
+                                      Shall we include the ruling pattern also for this record?-->
```

Comment:
maybe at the bottom of f. 10? But no, not really. Let's Massimo have a look as well.

Reply:
I can see horizontal lines at the bottom of f. 3v (pic 12) and 6r (pic 14), but most folia seem to have no ruled lines.